codegen-renderers/src/main/kotlin/io/vrap/codegen/languages/java/model/second/JavaObjectTypeClassRenderer.kt

```diff
@@ -0,0 +1,150 @@
+package io.vrap.codegen.languages.java.model.second
+
+import com.google.inject.Inject
+import io.vrap.codegen.languages.extensions.isPatternProperty
+import io.vrap.codegen.languages.java.JavaSubTemplates
+import io.vrap.codegen.languages.java.extensions.simpleName
+import io.vrap.codegen.languages.java.extensions.toComment
+import io.vrap.rmf.codegen.io.TemplateFile
+import io.vrap.rmf.codegen.rendring.utils.escapeAll
+import io.vrap.rmf.codegen.rendring.utils.keepIndentation
+import io.vrap.rmf.codegen.types.VrapObjectType
+import io.vrap.rmf.codegen.types.VrapTypeProvider
+import io.vrap.rmf.raml.model.types.ObjectType
+import io.vrap.rmf.raml.model.types.Property
+
+class JavaObjectTypeClassRenderer @Inject constructor(override val vrapTypeProvider: VrapTypeProvider) : JavaObjectTypeRenderer() {
+
+    override fun render(type: ObjectType): TemplateFile {
+
+        val vrapType = vrapTypeProvider.doSwitch(type) as VrapObjectType
+
+        val parentTypeImport = if(type.type != null && type.type is ObjectType) {
+            val parentType = vrapTypeProvider.doSwitch(type.type) as VrapObjectType
+            "${parentType.`package`}.${parentType.simpleClassName}"
+        }else{
+            null
+        }
+
+        val content = """
+                |package ${vrapType.`package`};
+                |
+                |${type.imports()}
+                |${parentTypeImport?.let { "import ${it}Impl;"} ?: ""}
+                |import com.fasterxml.jackson.annotation.*;
+                |import javax.annotation.Generated;
+                |import javax.validation.Valid;
+                |import javax.validation.constraints.NotNull;
+                |import java.util.*;
+                |import org.apache.commons.lang3.builder.EqualsBuilder;
+                |import org.apache.commons.lang3.builder.HashCodeBuilder;
+                |import org.apache.commons.lang3.builder.ToStringBuilder;
+                |import org.apache.commons.lang3.builder.ToStringStyle;
+                |import java.util.List;
+                |
+                |<${type.toComment().escapeAll()}>
+                |<${type.subTypesAnnotations()}>
```

Comment:
I think that we don't have to repeat these jackson annotations here and I would suggest to remove them.